wrappers/golang/README.md

```diff
@@ -26,7 +26,7 @@ If you wish to build for a specific curve, for example bn254, without G2 or ECNT
 ```
 
 >[!NOTE]
->Current supported curves are `bn254`, `bls12_381`, `bls12_377` and `bw6_671`
+>Current supported curves are `bn254`, `bls12_381`, `bls12_377`, `bw6_671` and `grumpkin`
```

Comment:
Should add current supported fields as well with Babybear support

Reply:
Fixed